engine/runtime/src/main/java/org/enso/interpreter/util/ScalaCollections.java

```diff
@@ -0,0 +1,21 @@
+package org.enso.interpreter.util;
+
+import java.util.Arrays;
+
+public final class ScalaCollections {
```

Comment:
There already is `ScalaConversions` and methods like `nil` and `cons`. Put your new method into that class.